src/pybind/mgr/dashboard/frontend/src/app/shared/api/upgrade.service.ts

```diff
@@ -57,6 +57,9 @@ export class UpgradeService extends ApiClient {
   }
 
   start(version?: string, image?: string) {
+    if (version === undefined) {
+      return this.http.post(`${this.baseURL}/start`, { image: image });
+    }
```

Comment:
i don't think this might be needed, since version is an `optional` parameter and undefined might not get through it?